cg/server/api.py

```diff
@@ -68,7 +67,18 @@
 @BLUEPRINT.route("/submit_order/<order_type>", methods=["POST"])
 def submit_order(order_type):
     """Submit an order for samples."""
-    api = OrdersAPI(lims=lims, status=db, osticket=osticket, freshdesk_client=freshdesk_client)
+    ticket_handler = TicketHandler(
```

Comment:
The `TicketHandler` should not be instantiated in the endpoint. Create it in the `ext.py` module instead and import it.

Reply:
Or rather, since it is only used by the `OrdersAPI`, you can just create the `TicketHandler` and then the `OrdersAPI` in the `ext.py` module and import it here.